src/app/widgets/cat-search/cat-search.component.html

```diff
@@ -8,4 +7,25 @@
     <option value="snrcat">SNRcat</option>
   </select>
 
-</p>
+
+
+</p> -->
+
+<div style="width:300px; margin-bottom:20px;" *ngIf="catalog">
+  <h3>Select a source:</h3>
+
+  <ng-select
+              [allowClear]='true'
+              [items]='items'
+              [disabled]='disabled'
+              (data)='refreshValue($event)'
+              (selected)='selected($event)'
+              (removed)='removed($event)'
+              (typed)='typed($event)'
+              placeholder='No source selected'>
```

Comment:
Maybe move the closing `>` to the next line to make it more obvious what's going on here?